packages/forms/test/template_integration_spec.ts

```diff
@@ -1967,6 +1967,58 @@ import {NgModelCustomComp, NgModelCustomWrapper} from './value_accessor_integrat
            expect(form.valid).toBeFalse();
            expect(form.controls.min_max.errors).toEqual({max: {max: -10, actual: 0}});
          }));
+
+      it('should call registerOnValidatorChange when formGroup input changes', fakeAsync(() => {
+           let registerOnValidatorFired = 0;
+
+           @Directive({
+             selector: '[ng-empty-custom-validator]',
+             providers: [{
+               provide: NG_VALIDATORS,
+               useExisting: forwardRef(() => EmptyCustomValidator),
+               multi: true
+             }]
+           })
+           class EmptyCustomValidator implements Validator {
+             @Input() injectedInput = '';
+
+             validate(c: AbstractControl) {
+               return null;
+             }
+
+             public registerOnValidatorChange(fn: () => void) {
+               registerOnValidatorFired++;
+             }
+           }
+
+           @Component({
+             selector: 'ng-model-empty-custom-validation',
+             template: `
+              <form>
+                <div ngModelGroup="emptyGroup" ng-empty-custom-validator [injectedInput]="ei.value">
+                  <input name="fgInput" #ei ngModel>
+                </div>
+              </form>
+            `
+           })
+           class NgModelEmptyCustomValidation {
+           }
+
+           const fixture = initTest(NgModelEmptyCustomValidation, EmptyCustomValidator);
+           fixture.detectChanges();
+           tick();
+
+           const input = fixture.debugElement.query(By.css('input')).nativeElement;
+           const form = fixture.debugElement.children[0].injector.get(NgForm);
+
+           input.value = 'foo';
+           dispatchEvent(input, 'input');
+           fixture.detectChanges();
+           expect(form.valid).toBeTruthy();
```

Comment:
nit: may be add a new line before the first expect here to visually separate asserts from the rest of the tests:
```suggestion
           fixture.detectChanges();
           
           expect(form.valid).toBeTruthy();
```